main/src/main/java/cgeo/geocaching/filters/gui/StoredListCheckboxFilterViewHolder.java

```diff
@@ -0,0 +1,85 @@
+package cgeo.geocaching.filters.gui;
+
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+import android.widget.LinearLayout;
+import android.widget.Space;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import cgeo.geocaching.R;
+import cgeo.geocaching.filters.core.IGeocacheFilter;
+import cgeo.geocaching.list.StoredList;
+import cgeo.geocaching.ui.TextParam;
+import cgeo.geocaching.ui.ViewUtils;
+
+public class StoredListCheckboxFilterViewHolder<T, F extends IGeocacheFilter> extends CheckboxFilterViewHolder {
+    public StoredListCheckboxFilterViewHolder(ValueGroupFilterAccessor filterAccessor) {
+        super(filterAccessor);
+    }
+
+    public StoredListCheckboxFilterViewHolder(ValueGroupFilterAccessor filterAccessor, int colCount, Set alwaysVisibleItems) {
+        super(filterAccessor, colCount, alwaysVisibleItems);
+    }
+
+    public View createView() {
```

Comment:
It should not be necessary to copy complete code of this method into subclass just to change a few lines. I recommend to abstract the superclass.better by introducing a new method e.g. to get the function to place on the button "add items", then just implement this method in the subclass. Currently the subclass contains must duplicated code, this should be avoided.